app/package.json

```diff
@@ -68,13 +68,14 @@
     },
     "dependencies": {
         "@babel/runtime-corejs2": "^7.3.1",
-        "@token-io/core": "^1.0.0",
+        "@token-io/core": "^1.0.1",
         "axios": "^0.19.0",
         "base64-js": "^1.3.0",
         "bs58": "4.0.0",
         "buffer": "^5.2.1",
         "fast-json-stable-stringify": "^2.0.0",
         "fast-sha256": "^1.1.0",
+        "lodash.merge": "^4.6.2",
```

Comment:
do we need this dependency? in general we should avoid adding dependencies

Reply:
Had to add this to resolve npm error, security vulnerability before